lib/name.ml

```diff
@@ -67,7 +67,7 @@ let parse_label base buf =
     | 0 ->
         Z base, 1
 
-    | v when ((v land 0b0_11000000) != 0) ->
+    | v when ((v land 0b0_11000000) == 0b0_11000000) ->
```

Comment:
I believe this can just be `=` since `==` is to test for physical equality (it shouldnt matter on integers)